spec/unit/view_spec.rb

```diff
@@ -173,4 +173,54 @@
     end
   end
 
+  context "templates" do
+
+    include GivenFilesystemSpecHelpers
+
+    use_given_filesystem
+
+    before(:each) do
+      s = Settings.new
+      mh = ManifestHandler.new s
+      mh.read_remote
+
+      v = View.new mh
+      v.templates = "one-column"
+
+      dir = given_directory
+
+      v.create(dir)
+      @path = File.join(dir, "index.html")
+    end
+
+    it "generates templates" do
+      expect(File.exists?(@path)).to be true
+    end
+
+    it "checks content" do
+      html_data = File.read(@path)
+
+      nokogiri_object = Nokogiri::HTML(html_data)
+      paragraphs_content = nokogiri_object.xpath("//p").to_s
+
+      expected_content =
+      '<p>
+        The goal of Inqlude is to provide a comprehensive listing of all
+        existing libraries for developers of Qt applications. If you are creating
+        applications using the <a href="http://qt-project.org">Qt toolkit</a>, and
+        are looking for libraries, components
+        or modules to use, Inqlude is the place where you find all information and
+        pointers to get started.
+      </p><p>
+        This is a young project, we are still collecting information, and are
+        building up the web site and the tools around it. If you would like to get
+        involved, read more about <a href="contribute.html">how to contribute</a>, or go
+        to the mailing list
+        <a href="https://mail.kde.org/mailman/listinfo/inqlude">inqlude.kde.org</a>
+        to directly talk to us. See you there.
+      </p>'
```

Comment:
This test is not ideal, because it depends a lot on the content of the concrete HTML template. The point of this test is not to test the content, though, but to test that the correct template is generated. It would be better to use fake data, which is set up by the test. This would need a bit of refactoring to be able to configure the template source path, thoiugh. I'll merge the pull request for now, so we can proceed, but maybe you can get back to this later and do the refactoring to improve the tests.

Another thing is that the tests have become quite slow. That also is something worth investigating and fixing.